avivator/src/constants.js

```diff
@@ -26,6 +26,6 @@ export const COLOR_PALLETE = [
   [255, 255, 255],
   [255, 0, 0]
 ];
-export const GLOBAL_SLIDER_DIMENSION_FIELDS = ['z', 'time'];
+export const GLOBAL_SLIDER_DIMENSION_FIELDS = ['z', 't'];
```

Comment:
I am a bit concerned that this is now within viv. Axis labels shouldn't be special, and now the code is sensitive to whatever they are named.

Reply:
I'm fine removing this.  It's only used for a default setting on the PIP and SBS viewers.  It's fine to require that people pass this in.